### PR TITLE
[TrilinosApplication] Clean up files in tests

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**

Clean up files in tests (ordering as in core)

**🆕 Changelog**

- [Clean up files in tests](https://github.com/KratosMultiphysics/Kratos/commit/a9f251018248574393c831f69930052d44b86d24)

### CODE REVIEW

File: applications/TrilinosApplication/tests/test_trilinos_redistance.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def setUp(self):
         self.model_part.AddNodalSolutionStepVariable(KratosMultiphysics.NODAL_AREA)
         self.model_part.AddNodalSolutionStepVariable(KratosMultiphysics.PARTITION_INDEX)
 
-        ReadDistributedModelPart(GetFilePath("coarse_sphere"), self.model_part)
+        ReadDistributedModelPart(GetFilePath(os.path.join("auxiliary_files", "mdpa_files", "coarse_sphere")), self.model_part)
```
Comment (philbucher) [inline applications/TrilinosApplication/tests/test_trilinos_redistance.py]: Please use pathlib.Path

**Review by philbucher — CHANGES_REQUESTED** [review]
Minor

File: applications/TrilinosApplication/tests/test_trilinos_redistance.py (hunk 1 above)
Comment (loumalouomega) [reply applications/TrilinosApplication/tests/test_trilinos_redistance.py]: Done

File: applications/TrilinosApplication/tests/test_trilinos_levelset_convection.py [hunk 2]
```diff
@@ -8,10 +9,6 @@
 from KratosMultiphysics.testing.utilities import ReadDistributedModelPart
 from KratosMultiphysics.TrilinosApplication import trilinos_linear_solver_factory
 
-
-def GetFilePath(fileName):
-    return os.path.join(os.path.dirname(os.path.realpath(__file__)), fileName)
```
Comment (philbucher) [inline applications/TrilinosApplication/tests/test_trilinos_levelset_convection.py:13]: this should be 
~~~
def GetFilePath(fileName):
    return pathlib.Path(__file__).absolute().parent / fileName
~~~

and so on ...

Comment (loumalouomega) [reply applications/TrilinosApplication/tests/test_trilinos_levelset_convection.py:13]: but here it is removed

File: applications/TrilinosApplication/tests/test_trilinos_levelset_convection.py [hunk 3]
```diff
@@ -29,14 +26,17 @@ def ConvectionVelocity(x, y, z):
     vel[0] = 1.0
     return vel
 
+def GetFilePath(fileName): 
+    return os.path.join(pathlib.Path(__file__).absolute().parent, fileName)
```
Comment (philbucher) [inline applications/TrilinosApplication/tests/test_trilinos_levelset_convection.py]:
```suggestion
    return pathlib.Path(__file__).absolute().parent / fileName

```

the goal is to get rid of `os.path` altogether

Comment (loumalouomega) [reply applications/TrilinosApplication/tests/test_trilinos_levelset_convection.py]: I tried this and didn't work, let me a second and I post the error

Comment (loumalouomega) [reply applications/TrilinosApplication/tests/test_trilinos_levelset_convection.py]: Okay, it is because it not a string, wait a sec

Comment (loumalouomega) [reply applications/TrilinosApplication/tests/test_trilinos_levelset_convection.py]: Now

**Review by philbucher — APPROVED** [review]
Thx

**Pull request merged**